README.md

```diff
@@ -46,18 +46,29 @@ By default, PCW runs on http://127.0.0.1:8000/
 
 ## Building a container
 
+Building your own container is not necessary anymore. You can use the ones already build as described in the next section.
```

Comment:
it is a bit odd to have section which started with words "you don't need this " :)  . I would move this to the end and make it as "Note: "

Reply:
I think you going too far here . I would stop where you just show what user CAN ( build container or user pre-build one ) so user can decide what he/she actually NEED :)